packages/rich-text/src/index.ts

```diff
@@ -38,4 +38,4 @@ export { default as __unstableFormatEdit } from './component/format-edit';
  * An object which represents a formatted string. See main `@wordpress/rich-text`
  * documentation for more information.
  */
-export { RichTextValue } from './types';
+export { type RichTextValue } from './types';
```

Comment:
The syntax `export { type X };` is highly untypical, although for some reason it works 😅 

Did you instead mean this, which will also work, but also look more familiar to everyone:

```suggestion
export type { RichTextValue } from './types';
```

Reply:
Thanks @tyxla!! It was intentional in case there were mixed exports. I've seen it in some linting rules, e.g, https://typescript-eslint.io/rules/consistent-type-exports/#fixmixedexportswithinlinetypespecifier but agree that it's not usual in Gutenberg. Cheers for the commit suggestion 👍